pkg/compiler/admin.go

```diff
@@ -0,0 +1,36 @@
+package compiler
+
+import (
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+)
+
+// This object is meant to satisfy github.com/lyft/flytepropeller/pkg/compiler/common.InterfaceProvider
```

Comment:
This comment does not answer why this is needed here?

Reply:
We need a struct to satisfy the interface because the function that we pass it to, `compiler.CompileWorkflow` takes the `common.InterfaceProvider` interface.